includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,503 @@
+<?php
+
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor playing card`s widget.
+ *
+ * Elementor widget playing cards.
+ *
+ * @since 1.0.0
+ */
+class Widget_Playing_Cards extends Widget_Base {
+
+	/**
+	 * Get widget name.
+	 *
+	 * Retrieve sidebar widget name.
+	 *
+	 * @return string Widget name.
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 */
+	public function get_name(): string {
+		return 'playing-cards';
+	}
+
+	/**
+	 * Get widget title.
+	 *
+	 * Retrieve sidebar widget title.
+	 *
+	 * @return string Widget title.
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 */
+	public function get_title(): string {
+		return __( 'Playing Cards', 'elementor' );
+	}
+
+	/**
+	 * Get widget icon.
+	 *
+	 * Retrieve playing cards widget icon.
+	 *
+	 * @return string Widget icon.
+	 * @since 3.12.1
+	 * @access public
+	 *
+	 */
+	public function get_icon(): string {
+		return 'eicon-heart';
+	}
+
+	/**
+	 * Get widget keywords.
+	 *
+	 * Retrieve the list of keywords the widget belongs to.
+	 *
+	 * @return array Widget keywords.
+	 * @since 2.1.0
+	 * @access public
+	 *
+	 */
+	public function get_keywords(): array {
+		return [ 'card', 'playing cards' ];
+	}
+
+	/**
+	 * Register playing-cards widget controls.
+	 *
+	 * Adds card`s and card styles.
+	 *
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function register_controls(): void {
+		$this->start_controls_section(
+			'cards_section',
+			[
+				'label' => __( 'Cards', 'elementor' ),
+				'tab' => Controls_Manager::TAB_CONTENT,
+			]
+		);
+
+		$repeater = new Repeater();
+
+		$repeater->add_control(
+			'card_number',
+			[
+				'label' => __( 'Card Number', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'A' => __( 'A', 'elementor' ),
+					'2' => __( '2', 'elementor' ),
+					'3' => __( '3', 'elementor' ),
+					'4' => __( '4', 'elementor' ),
+					'5' => __( '5', 'elementor' ),
+					'6' => __( '6', 'elementor' ),
+					'7' => __( '7', 'elementor' ),
+					'8' => __( '8', 'elementor' ),
+					'9' => __( '9', 'elementor' ),
+					'10' => __( '10', 'elementor' ),
+					'J' => __( 'J', 'elementor' ),
+					'Q' => __( 'Q', 'elementor' ),
+					'K' => __( 'K', 'elementor' ),
+				],
+				'default' => 'A',
+			]
+		);
+
+		$repeater->add_control(
+			'card_suit',
+			[
+				'label' => __( 'Card Suit', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'♥' => __( 'Hearts', 'elementor' ),
+					'♦' => __( 'Diamonds', 'elementor' ),
+					'♣' => __( 'Clubs', 'elementor' ),
+					'♠' => __( 'Spades', 'elementor' ),
+				],
+				'default' => '♥',
+			]
+		);
+
+		$repeater->add_control(
+			'card_background',
+			[
+				'label' => __( 'Card Background', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} {{CURRENT_ITEM}}.e-playing-card' => '--e-card-bg-color: {{VALUE}};',
```

Comment:
Usually targeting the wrapper selector is sufficient.


```suggestion
					'{{WRAPPER}}' => '--e-card-bg-color: {{VALUE}};',
```